app/assets/javascripts/utils/model_utils.js

```diff
@@ -61,3 +61,13 @@ export const sortByKey = (models, sortKey, previousKey = null, desc = false, abs
   }
   return { newModels, newKey };
 };
+
+// transformUsers: Transforms user objects by splitting 'real_name' into 'first_name' and 'last_name'
+export const transformUsers = (users) => {
```

Comment:
I suggest adding tests for this function, so we can show what it puts out for various types of real names as well as users without a real_name.

It should probably also return the original user object rather than a new one in cases where no real_name is present.

Reply:
Okay i will add the test